### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by siavoshqt3** [aptos-labs/aptos-core#9380]
Hello aptos team!
I have an issue with my aptos wallet(petra wallet)
I created a petra wallet about 10 month ago(exactly the day that aptos went mainnet)
I used my address and anything was OK... I imported my seed in martian and pontem wallet and use the those too...
After an update on my Google Chrome all my wallet reset...
I wanted to recover my aptos wallet but any time I entered my seed phrase a new address generated for me(different from my original address!)
I try to explain issue in petra discord,but...nothing happenned unfortunately!
With my research I knew that there is not a common method to generate address from one seed phrase!!!
I think problem is that for example in martian and pontem wallet many addresses can be generate from one seed phrase but in petra wallet there is just one unique address generate for one seed phrase!!!
So I want to request a new feature on petra wallet to solve this issue.
Im here if you want Any further information about this...

**Comment by kent-white** [comment]
hmm, @siavoshqt3 what happens when you import it to Pontem or Martian, they have a list of address' derived from the seed phrase?

**Comment by siavoshqt3** [comment]
Yes there is list of addresses drived from my seed phrase. And unfortunately my address is not among them! Could it be solve?

Sent from Yahoo Mail on Android 
 
  On Mon, Aug 7, 2023 at 20:42, Kent ***@***.***> wrote:   


hmm, @siavoshqt3 what happens when you import it to Pontem or Martian, they have a list of address' derived from the seed phrase?

**Comment by GeverGm** [comment]
Team! I also faced this problem, I hope you can help your users, because we have entrusted you with our funds

**Comment by amirr1360** [comment]
Hi everyone
I have this problem with my pontem wallet
I changed the Windows laptop and installed my pontem extension and imported 12 words
But my wallet address had changed
